src/edge/edgeManager.ts

```diff
@@ -345,21 +357,28 @@ export class EdgeManager {
             null, dftValue);
     }
 
-    private async inputImage(module: string, template: string): Promise<{ repositoryName: string, imageName: string }> {
+    private async inputImage(module: string, template: string): Promise<{ repositoryName: string, imageName: string, moduleTwin: object }> {
         let repositoryName: string = "";
         let imageName: string = "";
+        let moduleTwin: object = null;

```

Comment:
Minor comment: use undefined instead of null: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined